testdata/schema/oracle.sql

```diff
@@ -3,6 +3,11 @@ DECLARE
     C INT;
```

Comment:
Oracle support is no more in v3

This file should actually have been deleted, but it was just forgotten...

Reply:
Ok, I've removed the oracle.sql file.